content/en/blog/_posts/Use containerd to handle k8s.gcr.io deprecation

```diff
@@ -0,0 +1,174 @@
+---
+layout: blog
+title: "Use containerd to handle k8s.gcr.io deprecation"
+date: 
+slug: handle-registry-deprecation-w-containerd
+---
+
+# Use containerd to handle k8s.gcr.io deprecation
+
+The Kubernetes community is getting ready for yet another major change. Until fall of 2022, `k8s.gcr.io` container registry hosted many Kubernetes community-managed containers images like Cluster Autoscaler, metrics-server, cluster-proportional-autoscaler. The “gcr.io” is Google Cloud Registry. In order to be vendor neutral, the community is moving away from using Google Cloud Registry to host container images.
+
+As a result, Starting March 20th, traffic from the old `k8s.gcr.io` registry is being redirected to `registry.k8s.io`. The older `k8s.gcr.io` will remain functioning for sometime but it is eventually getting deprecated.
+
+![Image.jpeg](https://res.craft.do/user/full/9d54cc03-adfe-f72f-3389-565eb7356d1d/doc/13FE78CC-1DDD-47D9-B875-EE3FB15F916E/FC421666-8104-4CD9-8C7F-8E4F4BCE2B83_2/6olkWYEK1kl10toAonqYL40P1ZSQIUY2N6iHRwId78kz/Image.jpeg)
+
+## What’s the impact?
+
+The change that occurred six days after Pi day is unlikely to cause major problems. There are some edge cases. But unless you operate in an airgapped or highly restrictive environment that applies strict domain name access controls, you won’t notice the change.
+
+This doesn’t mean that there’s no action required. Now is the time to scan code repositories and clusters for usage of the old registry. Failing to act will result in cluster components failing.
+
+Once the old registry goes away, **Kubernetes will not be able to create new Pods** (unless image is cached) if the container uses an image hosted on `k8s.gcr.io`.
+
+## What do you need to change?
+
+Cluster owners and development teams have to ensure they are not using any images stored in the old registry. The change is fairly simple. It's pretty simple, really.
+
+You need to change your manifests to use `registry.k8s.io` container registry.
+
+![Image.png](https://res.craft.do/user/full/9d54cc03-adfe-f72f-3389-565eb7356d1d/doc/13FE78CC-1DDD-47D9-B875-EE3FB15F916E/EB0981CC-3DC8-49EE-A6F5-A857A69CC949_2/D0F0mQ2JMAl55JNCuS66lUylRznVZywEMdojiRqkc0sz/Image.png)
```

Comment:
1. Set an alt attribute that helps accessibility
2. Use SVG (or PNG if necessary). JPEG and friends are for photos.
3. Include the image in the commit itself. Don't reference an external image.

Reply:
Done.